xbmc/cores/dvdplayer/DVDInputStreams/DVDInputStreamBluray.cpp

```diff
@@ -558,7 +558,7 @@ void CDVDInputStreamBluray::ProcessEvent() {
     m_player->OnDVDNavResult((void*) &pid, 3);
     break;
 
-#if (BLURAY_VERSION >= BLURAY_VERSION_CODE(0,2,2))
+#if (BLURAY_VERSION > BLURAY_VERSION_CODE(0,2,2))
```

Comment:
right. i have squashed an unrelated commit. but the change is correct, BD_EVENT_MENU was exposed in 0.2.3 and is not around in 0.2.2